### PR TITLE
fix: solve persist tokens issue

### PR DESCRIPTION
**Pull request by lucasgabrielgsp**
- [x] solve persist tokens issue

### CODE REVIEW

File: source/pages/Tokens/CustomToken.tsx [hunk 1]
```diff
@@ -47,13 +49,17 @@ export const CustomToken = () => {
         web3Provider
       );
 
+      const balance = ethers.utils.formatEther(metadata.balance);
```
Comment (claudiovb) [inline source/pages/Tokens/CustomToken.tsx:52]: We need to format the token by the amount of decimals the token has, only for blockchain native tokens it should be used format either function

**Review by claudiovb — CHANGES_REQUESTED** [review]
Change the formation for general tokens

File: source/pages/Tokens/CustomToken.tsx (hunk 1 above)
Comment (lucasgabrielgsp) [reply source/pages/Tokens/CustomToken.tsx:52]: Done. Here is the comparation between metamask and pali: 
Screen Shot 2022-11-15 at 12 07 29
Screen Shot 2022-11-15 at 12 08 54

**Review by claudiovb — COMMENTED** [review]
Just missing to add watchMemPool back on reset

**Review by claudiovb — APPROVED** [review]

**Pull request merged**